fastapi/applications.py

```diff
@@ -1788,6 +1792,14 @@ def get(
                 """
             ),
         ] = Default(generate_unique_id),
+        add_auto_options_route: Annotated[
+            Optional[bool],
+            Doc(
+                """
+                Auto create options route.
```

Comment:
How about something like this?

```suggestion
                Automatically create a route to handle the OPTIONS HTTP verb.
```

Reply:
thanks, updated